front/lib/api/assistant/actions/tables_query.ts

```diff
@@ -240,6 +218,65 @@ export class TablesQueryConfigurationServerRunner extends BaseActionConfiguratio
       }),
     };
 
+    // Render conversation for the action.
+    const supportedModel = getSupportedModelConfig(agentConfiguration.model);
+    if (!supportedModel) {
+      yield {
+        type: "tables_query_error",
+        created: Date.now(),
+        configurationId: agentConfiguration.sId,
+        messageId: agentMessage.sId,
+        error: {
+          code: "tables_query_error",
+          message: `Error running TablesQuery app: model not supported.`,
+        },
+      };
+
+      throw new Error("Unreachable: Supported model not found.");
+    }
+
+    const allowedTokenCount =
+      supportedModel.contextSize - TABLES_QUERY_MIN_TOKEN;
+    if (allowedTokenCount < RENDERED_CONVERSATION_MIN_TOKEN) {
+      yield {
+        type: "tables_query_error",
+        created: Date.now(),
+        configurationId: agentConfiguration.sId,
+        messageId: agentMessage.sId,
+        error: {
+          code: "tables_query_error",
+          message: `Error running TablesQuery app: model context too small.`,
+        },
+      };
+
+      throw new Error(
```

Comment:
We should return here otherwise it's a stream dropping error?

Reply:
or if unreachable then no need to yield this will just crash the run anyway to throw right after.